packaging/docker/run.sh

```diff
@@ -34,18 +34,6 @@ create_group_and_assign_to_user() {
 	fi
 }
 
-DOCKER_USR="netdata"
-DOCKER_SOCKET="/var/run/docker.sock"
-DOCKER_GROUP="docker"
-
-if [ -S "${DOCKER_SOCKET}" ] && [ -n "${PGID}" ]; then
-	GRP=$(create_group_and_assign_to_user "${DOCKER_GROUP}" "${PGID}" "${DOCKER_USR}")
```

Comment:
Since you removed `create_group_and_assign_to_user`, kindly remove the declaration of that function above, we do not use it elsewhere